### PR TITLE
Added support for Tauri in the frontend.

### PR DESCRIPTION
**Pull request by AbooMinister25**
Integrated Tauri with the frontend, React app can now be used on the desktop. We can edit the react app as normal as if nothing changed, just do `yarn tauri dev` to run it, it'll take some time to build for the first time, but after that it's pretty quick.

### CODE REVIEW

**Review by Kronifer — APPROVED** [review]
mmm, njes. looks hgood.

File: frontend/package.json [hunk 1]
```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "dev": "webpack serve --mode=development",
     "build": "cross-env NODE_ENV=production webpack --mode production",
-    "lint": "eslint --ignore-path .gitignore --fix ."
+    "lint": "eslint --ignore-path .gitignore --fix .",
+    "tauri": "tauri",
```
Comment (dawnofmidnight) [inline frontend/package.json:9]: This line is unnecessary.

**Review by dawnofmidnight — CHANGES_REQUESTED** [review]
Seems pretty good, just this one thing.

**Comment by AbooMinister25** [comment]
> Seems pretty good, just this one thing.

Alright, I fixed the issue

**Review by dawnofmidnight — APPROVED** [review]

**Pull request merged**